quic/s2n-quic-core/src/packet/interceptor.rs

```diff
@@ -185,3 +203,31 @@ where
         self.tx.havoc(&mut self.random, payload);
     }
 }
+
+impl<T: Interceptor> Interceptor for Option<T> {
```

Comment:
This needs a `rx_remote_port` impl

Reply:
Does `impl<A, B> Interceptor for (A, B)` need one too?

Reply:
yep